### ISSUE TITLE
Dragon Roost/Den generation broken

### ISSUE THREAD

**Issue by ImpulseTM** [Alex-the-666/Ice_and_Fire#1902]
I have no idea what happened with this, I installed the following mods, Bewitchment, Howling moon, Mo Creatures, Vampirism. After this dragons completely stopped spawning, I have had Ice and Fire long before I added any of these mods and there have been no issues yet. After this happened I attempted to troubleshoot, uninstalled all the mods, their configs and content and sadly the dragons still do not spawn. I even did a fresh reinstall of Ice and Fire + the new 1.8.2 update and there is still no dragon spawns. There may be a compatibility issue or an outright problem. I will update as the situation unfolds, this may need extra help to figure this out.

**Comment by ImpulseTM** [comment]
-Note I had been looking into this for a good 20-30 minutes before posting and still have not yet found the issue and it makes me very sad because dragons will no longer spawn anywhere, including the twilight forest which I have had spawn a lot in the past with above ground dragon dens(because the surface height is at y30+).

**Comment by ImpulseTM** [comment]
Confirmed: Dragon lairs and Roosts do indeed spawn in the twilight forest now.

Speculation: Reinstalling forge may have fixed the issue with some files conflicting or otherwise. I noted in the menu as well when hovering over a config option other config descriptors would appear over it and overlap. i will continue testing in order to find out what mod caused dragon spawns to break.

Update #2: I reinstalled both MoCreatures + BeWitchment(And Patchouli scripts) and can confirm neither of these mods caused a generation issue. Final conclusion: Howling moon or Vampirism caused a bug with the dragon generation, this may need to be looked into for future issues.
